modules/packages/composer/metadata.go

```diff
@@ -44,7 +51,7 @@ type Metadata struct {
 	Description string            `json:"description,omitempty"`
 	Readme      string            `json:"readme,omitempty"`
 	Keywords    []string          `json:"keywords,omitempty"`
-	Comments    Comments          `json:"_comments,omitempty"`
+	Comments    Comments          `json:"_comment,omitempty"`
```

Comment:
Here is a bug. It should be `_comment`

https://getcomposer.org/doc/04-schema.md#-comment